server/src/test/java/org/elasticsearch/env/NodeRepurposeCommandTests.java

```diff
@@ -63,7 +63,7 @@ public void createNodePaths() throws IOException {
         try (NodeEnvironment nodeEnvironment = new NodeEnvironment(dataMasterSettings, environment)) {
             nodePaths = new Path[] { nodeEnvironment.nodeDataPath() };
             final String nodeId = randomAlphaOfLength(10);
-            try (PersistedClusterStateService.Writer writer = new PersistedClusterStateService(nodePaths, nodeId,
+            try (PersistedClusterStateService.Writer writer = new PersistedClusterStateService(nodePaths[0], nodeId,
```

Comment:
We can fix `nodePaths` here too, indeed it's only used in this one method so can just be a local variable of type `Path`.